CHANGELOG.md

```diff
@@ -0,0 +1,66 @@
+    * `--allow-setuid`: Allows setuid binaries to be mounted into the container (requires root)
+    * `--apply-cgroups <path>`: Apply cgroups configuration from file to contained processes (requires root)
+    * `--dns <string>`: Adds the comma separated list of DNS servers to the containers `resolv.conf` file
+    * `--drop-caps <string>`: Drop the specified capabilities from the container (requires root)
+    * `--fakeroot`: Run the container in a user namespace as `uid=0`. Requires a recent kernel to function properly
+    * `--hostname <string>`: Set the hostname of the container
+    * `--keep-privs`: Keep root user privilege inside the container (requires root)
+    * `--network <string>`: Specify a list of comma separated network types ([CNI Plugins](https://github.com/containernetworking/cni)) to be present inside the container, each with its own dedicated interface in the container
+    * `--network-args <string>`: Specify arguments to pass to CNI network plugins (set by `--network`)
+    * `--no-privs`: Drop all privileges from root user inside the container (requires root)
+    * `--security <string>`: Configure security features such as SELinux, Apparmor, Seccomp...
+    * `--writable-tmpfs`: Run container with a `tmpfs` overlay
+  * The command `singularity instance start` now supports the `--boot` flag to boot the container via `/sbin/init`
+  * Changes to image mounting behavior:
+    * All image formats are mounted as read only by default
+    * `--writable` only works on images which can be mounted in read/write [applicable to: `sandbox` and legacy `ext3` images]
+    * `--writable-tmpfs` runs the container with a writable `tmpfs`-based overlay [applicable to: all image formats]
+    * `--overlay <string>` now specifies a list of `ext3`/`sandbox` images which are set as the containers overlay [applicable to: all image formats] 
+
+## Build Command Changes:
+  * All images are now built as [Singularity Image Format (SIF)](https://www.sylabs.io/2018/03/sif-containing-your-containers/) images by default
+  * When building to a path that already exists, `singularity build` will now prompt the user if they wish to overwrite the file existing at the specified location
+  * The `-F|--force` flag now overrides the interactive prompt and will always attempt to overwrite the file existing at the specified location
+  * The `-u|--update` flag has been added to support the workflow of running a definition file on top of an existing container [implies `--sandbox`, only supports `sandbox` image types]
+  * The `singularity build` command now supports the following flags for integration with the [Sylabs.io Cloud Library](https://cloud.sylabs.io/library):
+    * `-r|--remote`: Build the image remotely on the Sylabs Remote Builder (currently unavailable)
+    * `-d|--detached`: Detach from the `stdout` of the remote build [requires `--remote`]
```

Comment:
We should change this so that it simple assumes remote instead of requiring that a user passes both.

Reply:
Well, that's not the behavior of this flag right now and we most likely will not be changing that for the time being so it should stay this way for now

Reply:
Right.  I mean we should change behaviour of the flag.